### PR TITLE
Zone.strong

### PR DESCRIPTION
**Pull request by floitschG**
The `dart:async` zone API is in the process of becoming strong-mode clean.
This patch fixes a breakage introduced by this change.

The change should have no semantic difference, even when running with the non-strong zone API.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: test
-version: 0.12.24+5
+version: 0.12.24+6
```
Comment (grouma) [inline pubspec.yaml]: Do we need to update the minimum sdk version as well?

Comment (floitschG) [reply pubspec.yaml]: No. This change should maintain the same semantics.

**Comment by floitschG** [comment]
I added a Changelog entry.

**Comment by nex3** [comment]
@floitschG It looks like as of 2.0.0-dev.1.0, `test` is still experiencing a bunch of Zone-related errors ([for example](https://travis-ci.org/dart-lang/test/jobs/277993772#L1056-L1095)). Are there further changes needed?

**Pull request merged**